README.md

```diff
@@ -106,6 +106,8 @@ The most basic tag type is a simple variable. A `{{name}}` tag renders the value
 
 All variables are HTML-escaped by default. If you want to render unescaped HTML, use the triple mustache: `{{{name}}}`. You can also use `&` to unescape a variable.
 
+To print `{{name}}` as a string literal, the default delimiters must be changed. See the ["Set Delimiter'](https://github.com/janl/mustache.js#set-delimiter) section for more information about custom delimiters.
+
```

Comment:
I believe that @bobthecow is right in objecting to the use of "string literal" in this case. I'd suggest avoiding the use of the word "literal" altogether because (1) "literal" has special significance in computer language jargon, which isn't necessary here, and (2) unfortunately, in modern English the words "literal" and "literally" often mean their opposites, "figurative" and "figuratively." 

How about this?

If you want `{{name}}` *not* to be interpreted as a mustache tag and to appear simply as `{{name}}` in the output, you must change and then restore the default delimiter.